### PR TITLE
React Static List of Posts

### PR DESCRIPTION
**Pull request by Hamuud**
[DEMO LINK](https://Hamuud.github.io/react_static-list-of-posts/)

### CODE REVIEW

File: src/components/CommentInfo/CommentInfo.tsx [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+      >
+        {comment.email}
+      </a>
+    </div>
+
+    <div className="CommentInfo__body">
+      {comment.body}
+    </div>
+  </div>
 );
```
Comment (Olekk17) [inline src/components/CommentInfo/CommentInfo.tsx:27]: U can use destructuring here, but I don't think that this is a problem at all

**Review by Olekk17 — APPROVED** [review]
Great, nice job

File: src/App.tsx [hunk 2]
```diff
@@ -2,108 +2,42 @@ import React from 'react';
...
+    ...post,
+    user: getUserById(post.userId),
+    comments: getCommentsById(post.id),
+  }
+  ));
+
+  return (
+    <section className="App">
+      <h1 className="App__title">Static list of posts</h1>
+      <PostList posts={posts} />
```
Comment (jstmpelowycz) [inline src/App.tsx:40]:
```suggestion
      <h1 className="App__title">
        Static list of posts
      </h1>

      <PostList posts={posts} />
```

File: src/components/CommentInfo/CommentInfo.tsx [hunk 3]
```diff
@@ -1,5 +1,27 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+type Props = {
+  comment: Comment;
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => (
```
Comment (jstmpelowycz) [inline src/components/CommentInfo/CommentInfo.tsx:8]: don't forget to destructure complex props

**Review by jstmpelowycz — APPROVED** [review]

**Review by jstmpelowycz — APPROVED** [review]

**Pull request closed without merging**